public/css/history.css

```diff
@@ -1,12 +1,23 @@
 .wrapper {
-	padding-top: 130px;
-	background-color: #fafafa;
-	color: #2b2b2b;
+	padding-top: 66px;
+}
+
+#history_logo {
+	position: absolute;
+	top: 170px;
+	left: 50%;
+	transform: translateX(-50%);
+	max-width: 100%;
+}
```

Comment:
Also on small screens, the logo will overlap with the mountains, making it hard to read. Reduce the `top` style for small screens to something like 120px so it overlaps less.